api/v1alpha3/domachine_types.go

```diff
@@ -57,6 +57,10 @@ type DOMachineStatus struct {
 	// Addresses contains the DigitalOcean droplet associated addresses.
 	Addresses []corev1.NodeAddress `json:"addresses,omitempty"`
 
+	// Volumes contains the DigitalOcean droplet associated block storage
```

Comment:
nit: I feel like this comment can fit in one line. It's not longer than the comment below it even when we don't wrap. This suggestion could be applied to other similar comments in this PR. Up to you though, this is not a blocker and purely preference on my part.

Reply:
The majority of other comments even on that type seem to wrap at 80 chars max which is pretty common.
The comment below seems to be the exception.

Reply:
Ahh ok, fair enough